openfisca_uk/variables/finance/income.py

```diff
@@ -77,6 +77,21 @@ class hours_worked(Variable):
     definition_period = YEAR
 
 
+class in_work(Variable):
+    value_type = bool
+    entity = Person
+    label = u"Worked some hours"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        return (person("hours_worked", period) > 0) | (
```

Comment:
suggest breaking it up
```suggestion
        has_hours_worked = person("hours_worked", period) > 0
        has_emp_self_emp_income = add(person, period, ["employment_income", "self_employment_income"]) > 0
        return has_hours_worked | has_emp_self_emp_income
```

Reply:
Yeah, that is better